### ISSUE TITLE
Error in shinytheme(theme):

### ISSUE THREAD

**Issue by GakenouOF** [DillonHammill/DataEditR#46]
I have just installed the package and i tried using the `data_edit` function on my dataset. It keeps returning this error

_Error in shinytheme(theme):
    yeti is not an available theme. Valid themes are: ._

![Screenshot 2022-03-01 154520](https://user-images.githubusercontent.com/48384837/156180330-4c8f280a-e0c3-4bd4-8527-5807fde6d7b4.jpg)

PS: This package worked fine on a computer i was using previously. I am using a new one and i get this error message. I am not sure if it is a package issue or computer.

**Comment by DillonHammill** [comment]
Have you tried using a different theme?
```{r}
data_edit(mtcars, theme = "cosmo")
```

**Comment by GakenouOF** [comment]
I just tried it, say it is not an available theme

**Comment by GakenouOF** [comment]
I am using the latest version (1.2.0)

**Comment by drpturner** [comment]
Same for me unfortunately (Win11, R 4.1.0, RStudio 2022.02.0 Build 443, shinythemes 1.2.0).
Other shiny apps working as expected

**Comment by DillonHammill** [comment]
Looks like `shinythemes` has been depreciated in favour of `bslib` and this has recently changed in `shiny` as well. I tried upgrading to `bslib` but it causes some issue with the UI - I wil need to fiddle around with this a bit to get it to work.

In the meantime, I have just pushed an update to GitHub to bypass setting a theme when `theme = NULL`. 
```{r}
data_edit(mtcars, theme = NULL)
```

**Comment by drpturner** [comment]
Thanks very much for the quick response - this update works for me

**Comment by DillonHammill** [comment]
I just pushed an update to GitHub to make the switch to `bslib`. Unfortunately, using bootstrap 4 causes the `Done` button to move over to the left so I have decided to stick to bootstrap 3 for now. Also users can now create their own custom `bs_theme` and pass directly to the `theme` argument:
```{r}
custom_thme <- bs_theme(bootswatch = "cosmo")
data_edit(mtcars, theme = custom_theme)
```
Users can still simply supply the name of the theme to the `theme` argument as well:
```{r}
data_edit(mtcars, theme = "cosmo")
```

Thanks for reporting this issue. I will push this update to CRAN today.
